### ISSUE TITLE
Show list when user click in textfield

### ISSUE THREAD

**Issue by SilviaCebrian** [alexrindone/flutter_textfield_search#28]
Is there a way to display the list by focusing on the textfield without having to write on it? Thx

**Comment by alexrindone** [comment]
@SilviaCebrian that could be built into this library fairly easily...it's basically being able to scroll through the list of options when you focus on the textfield...is that accurate? If so I should be able to add that later tonight and publish a new release.

**Comment by SilviaCebrian** [comment]
Yes, that's it. I think it would be interesting to at least have the option to activate it. Thx!

**Comment by isurugg** [comment]
@alexrindone did you enable this option?

**Comment by Donly-Wilson** [comment]
Adding this parameter does the trick - minStringLength: -1,

**Comment by nemanja-nisic** [comment]
@Donly-Wilson does not work for me, probably got lost in some update
